helm/prometheus-agent-app/templates/_helpers.tpl

```diff
@@ -36,3 +36,17 @@ giantswarm.io/service-type: {{ .Values.serviceType }}
 helm.sh/chart: {{ include "chart" . | quote }}
 {{- end -}}
 
+{{/*
+prometheus-image
+*/}}
+{{- define "prometheus-image" -}}
+{{- printf "%s:%s" .Values.image .Chart.AppVersion | trunc 63 | trimSuffix "-" -}}
```

Comment:
Why not define it in the prometheus CR as this is only used there ? 

I'd use .Values.image.tag | default .Chart.AppVersion btw :D